### PR TITLE
add view evaluation template screen

### PR DESCRIPTION
**Pull request by emmaacha**
Ticket ID: [Create View Evaluation Template Screen #457](https://github.com/nerubia/kh360-react/issues/457)

BE PR: [update apis for view evaluation template screen #421](https://github.com/nerubia/kh360-nodejs/pull/421)

### CODE REVIEW

**Review by jatnerubia — CHANGES_REQUESTED** [review]
just a few changes ^^

**Comment by emmaacha** [comment]
Done ^^

File: src/features/admin/evaluation-templates/[id]/view-evaluation-template-header.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+import { getEvaluationTemplate } from "../../../../redux/slices/evaluation-template-slice"
+import { Checkbox } from "../../../../components/ui/checkbox/checkbox"
+
+export const ViewEvaluationTemplateHeader = () => {
+  const { id } = useParams()
+  const appDispatch = useAppDispatch()
+  const { evaluation_template } = useAppSelector((state) => state.evaluationTemplate)
+  const [isChecked, setIsChecked] = useState(false)
+
+  const handleChange = () => {
```
[the remark this answers is not in the document]
Comment (jatnerubia) [reply src/features/admin/evaluation-templates/[id]/view-evaluation-template-header.tsx:15]: 🤔

**Review by jatnerubia — CHANGES_REQUESTED** [review]
^^

File: src/features/admin/evaluation-templates/[id]/view-evaluation-template-header.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (emmaacha) [reply src/features/admin/evaluation-templates/[id]/view-evaluation-template-header.tsx]: Oh, my bad

[the remark this answers is not in the document]
Comment (emmaacha) [reply src/features/admin/evaluation-templates/[id]/view-evaluation-template-header.tsx]: Done ^^

**Review by jatnerubia — APPROVED** [review]
Cool

**Review by Dev-Nino — APPROVED** [review]

**Pull request merged**